integration/busybox.py

```diff
@@ -80,7 +80,9 @@ def create_tarball(self, split=False):
         for path in busybox.stdout.read().split("\n"):
             if not path.strip():
```

Comment:
you could `path = path.strip()` before this, and avoid a bunch of `strip()` below